xray_vision/qt_widgets/__init__.py

```diff
@@ -100,6 +101,17 @@ def __init__(self, data_list, key_list,
         # trigger the image to draw
         self._messenger.sl_update_image(0)
 
+        self.key_list = key_list
+
+        def add_data(self, new_data, new_key_names):
```

Comment:
Doc strings in this would be appreciated. Please use the numpy doc string format.